Updated README (#3)
filecoin-shipyard/shipyard: README.md

# Filecoin Shipyard

> The Filecoin Shipyard is a venue for the community to collaborate on code libraries, research experiments, products and more. It's an area for users from all walks to contribute their talents to create tools that continue to keep the Filecoin ecosystem robust and thriving. 

Reviewer: The phrase "from all walks" feels odd to me as a native English speaker without the "of life" attached. (I asked a friend and they had the same gut reaction.) However, I'd actually recommend picking a different phrase because I assume non-native English speakers wouldn't intuitively understand this expression. Maybe "of all backgrounds"?

Author: Changed :)

#### Have a project to add?

To propose or discuss a new Shipyard project, please create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) in this repository with as much information and as many relevant links as possible. To all existing contributors, thank you for your hard work!

## Our current residents

> Below are tables of the tools currently existing within the [Filecoin Shipyard](https://github.com/filecoin-shipyard). 

Reviewer: ```suggestion
The following tools, sorted by category, are those that currently reside within the [Filecoin Shipyard](https://github.com/filecoin-shipyard). 
```

| [js-lotus-client-rpc](https://github.com/filecoin-shipyard/js-lotus-client) | Lotus JS API JSON-RPC query module. |
| [js-lotus-client-provider-browser](https://github.com/filecoin-shipyard/js-lotus-client) | Lotus JS client web browser Provider module. |
| [tour-de-lotus](https://github.com/filecoin-shipyard/tour-de-lotus) | A slideshow tour of Lotus using the Lotus JS client. |
| [js-lotus-client-testground](https://github.com/filecoin-shipyard/js-lotus-client) | Tests for the Lotus JS API for use with Testground. |
| [rl1-lotus-workshop](https://github.com/filecoin-shipyard/rl1-lotus-workshop) | A workship for Ready Layer One.
| [npm-go-filecoin-dep](https://github.com/filecoin-shipyard/npm-go-filecoin-dep) | Download `go-filecoin` to a `node_modules` directory. |

### Metrics tools
| Name | Description  |
|---|---|
| [hactar](https://github.com/filecoin-shipyard/hactar) | Analysis and monitoring tool for Filecoin miners. |
| [telegraf-lotus](https://github.com/filecoin-shipyard/telegraf-lotus) | An analytics capture plugin for the Lotus daemon alongside Telegraf. |

### Wallets
| Name | Description  |
|---|---|

> | [filecoin-wallet](https://github.com/filecoin-shipyard/filecoin-wallet) A web-based Filecoin wallet implementation written in Java.

Reviewer: ```suggestion
| [filecoin-wallet](https://github.com/filecoin-shipyard/filecoin-wallet) | A web-based Filecoin wallet implementation written in Java. |
```

# Filecoin Shipyard

The Filecoin Shipyard is a venue for the community to collaborate on code libraries, research experiments, products and more. It's an area for people of all backgrounds to contribute their talents to help create some of the user-driven tools that continue to keep the Filecoin ecosystem robust and thriving. 

#### Have a project to add?

> To propose or discuss a new Shipyard project, please create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) in this repository with as much information and as many relevant links as possible. To all existing contributors, thank you for your hard work!

Reviewer 2: ```suggestion
To add an existing project to this listing, please create an create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) with the project Github link, description, and status.

To propose a brand new Shipyard repo, please create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) in this repository with as much information and as many relevant links as possible.

To all existing contributors, thank you for your hard work!
```

# Filecoin Shipyard

> The Filecoin Shipyard is a venue for the community to collaborate on code libraries, research experiments, products and more. It's an area for people of all backgrounds to contribute their talents to help create some of the user-driven tools that continue to keep the Filecoin ecosystem robust and thriving. 

Reviewer 2: ```suggestion
Welcome to the Filecoin Shipyard! Inspired by the active and bustling places where ships are built and repaired, this is a venue for the Filecoin community to pursue and collaborate on code libraries, tools, experiments, products, and more.

It serves as a companion to the [filecoin-project](https://github.com/filecoin-project) Github org.
```

> #### Have a project to add?

Reviewer 2: Let's move this section to the end, since it won't be relevant for most users.

To add an existing project to this listing, please create an create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) with the project Github link, description, and status.

To propose a brand new Shipyard repo, please create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) in this repository with as much information and as many relevant links as possible.

To all existing contributors, thank you for your hard work!

## Our current residents

> The following tools, sorted by category, are those that currently reside within the [Filecoin Shipyard](https://github.com/filecoin-shipyard).

Reviewer 2: Some of these tools don't reside within the Shipyard (and what we're looking for is an index of _all_ community projects, not important to filter out the ones residing elsewhere). Can you modify "residents" & "reside within" to something more expansive?

### API tools

| Name | Description  | Status |
|---|---|---|

> | [js-filecoin-api-client](https://github.com/filecoin-shipyard/js-filecoin-api-client) | A Filecoin API client implemented in JavaScript | Active |

Reviewer 2: Probably alphabetize them by repo name

| [java-filecoin-api-client](https://github.com/filecoin-shipyard/java-filecoin-api-client) |  A Filecoin API client implemented in Java | Active |
| [use-filecoin-config](https://github.com/filecoin-shipyard/use-filecoin-config) | React hook to load config via Filecoin API | Active |
| [use-filecoin-network-info](https://github.com/filecoin-shipyard/use-filecoin-network-info) | React hook to query network info via Filecoin API | Active |
| [use-filecoin-head](https://github.com/filecoin-shipyard/use-filecoin-head) | React hook to query head blocks and chain height via Filecoin API |  Active |
| [starling-api](https://github.com/smalldata-industries/starling-api) | A RESTful API that simplifies storing data |

### Block explorers

| Name | Description  | Status |
|---|---|---|

> | [filscan-block-explorer](https://github.com/filecoin-shipyard/filscan-block-explorer) | The Filscan.io block explorer | Active |

Reviewer 2: Also alphabetize

| [filecoin-block-explorer](https://github.com/filecoin-shipyard/filecoin-block-explorer) | A Filecoin block explorer | Active |

## Client tools

| Name | Description  | Status |
|---|---|---|
| [js-lotus-client](https://github.com/filecoin-shipyard/js-lotus-client) | Lotus JS client | Active |
| [js-lotus-client-schema](https://github.com/filecoin-shipyard/js-lotus-client-schema) | Lotus JS client schema model | Active |
| [js-lotus-client-rpc](https://github.com/filecoin-shipyard/js-lotus-client) | Lotus JS API JSON-RPC query module | Active |
| [js-lotus-client-provider-browser](https://github.com/filecoin-shipyard/js-lotus-client) | Lotus JS client web browser Provider module | Active |

> | [tour-de-lotus](https://github.com/filecoin-shipyard/tour-de-lotus) | A slideshow tour of Lotus using the Lotus JS client | Active |

Reviewer 2: Move to new Demos and Tutorials section

> | [js-lotus-client-testground](https://github.com/filecoin-shipyard/js-lotus-client) | Tests for the Lotus JS API for use with Testground | Active |

Reviewer 2: Move this one to an Infrastructure section

> | [js-lotus-client-testground](https://github.com/filecoin-shipyard/js-lotus-client) | Tests for the Lotus JS API for use with Testground | Active |

Reviewer 2: ```suggestion
| [js-lotus-client-testground](https://github.com/filecoin-shipyard/js-lotus-client-testground) | Tests for the Lotus JS API for use with Testground | Active |
```

> | [rl1-lotus-workshop](https://github.com/filecoin-shipyard/rl1-lotus-workshop) | A workship for Ready Layer One | Active |

Reviewer 2: Move to new Demos and Tutorials section

> | [rl1-lotus-workshop](https://github.com/filecoin-shipyard/rl1-lotus-workshop) | A workship for Ready Layer One | Active |

Reviewer 2: ```suggestion
| [rl1-lotus-workshop](https://github.com/filecoin-shipyard/rl1-lotus-workshop) | A workshop for Ready Layer One | Active |
```

> | [npm-go-filecoin-dep](https://github.com/filecoin-shipyard/npm-go-filecoin-dep) | Scripts for downloading `go-filecoin` to a `node_modules` directory | Active |

Reviewer 2: ```suggestion
| [npm-go-filecoin-dep](https://github.com/filecoin-shipyard/npm-go-filecoin-dep) | Scripts for downloading `go-filecoin` to a `node_modules` directory | Inactive |
```

> ### Miner tools

Reviewer 2: Despite the mining-evocative name, these are all demo apps ~and should be listed under the proposed Demos and Tutorials section. Check this thread if they should be marked as Deprecated. https://filecoinproject.slack.com/archives/CRK2LKYHW/p1591757081108700~.

They are deprecated. @jimpick is going to move them to his personal repo for now, so we can remove from this listing.

| [filecoin-pickaxe-direct deal](https://github.com/filecoin-shipyard/filecoin-pickaxe-direct-deal) | Filecoin deals for pickaxe bundles | Active |
| [filecoin-pickaxe-agent](https://github.com/filecoin-shipyard/filecoin-pickaxe-agent) | An agent for performing requested Filecoin tasks | Active |
| [filecoin-browse-asks](https://github.com/filecoin-shipyard/filecoin-pickaxe-asks) | A tool for scrolling through Filecoin asks in a terminal | Active |
| [filecoin-big-head](https://github.com/filecoin-shipyard/filecoin-big-head) | A tool for displaying current chain height in a terminal | Active |

### Miner metrics tools

| Name | Description  | Status |
|---|---|---|
| [hactar](https://github.com/filecoin-shipyard/hactar) | Analysis and monitoring tool for Filecoin miners | Active |
| [telegraf-lotus](https://github.com/filecoin-shipyard/telegraf-lotus) | An analytics capture plugin for the Lotus daemon alongside Telegraf |  Active |

### Wallets

| Name | Description  | Status |
|---|---|---|
| [filecoin-wallet](https://github.com/filecoin-shipyard/filecoin-wallet) | A web-based Filecoin wallet implementation written in Java | Active |

> | [filecoin-web-wallet](https://github.com/openworklabs/filecoin-web-wallet) | A web wallet built with Next.js that supports sending & receiving FIL from a Ledger device | **In progress** | 

Reviewer 2: ```suggestion
| [filecoin-web-wallet](https://github.com/openworklabs/filecoin-web-wallet) | The Glif web wallet. Built with Next.js and supports sending & receiving FIL from a Ledger device | **In progress** | 
```

> | [filecojn-rs](https://github.com/zondax/filecoin-rs) | Filecoin Signing Library - Rust / WASM / JSONRPC Signing Service | **In progress** |

Reviewer 2: ```suggestion
| [filecoin-rs](https://github.com/zondax/filecoin-rs) | Filecoin Signing Library - Rust / WASM / JSONRPC Signing Service | **In progress** |
```

> | [filecojn-rs](https://github.com/zondax/filecoin-rs) | Filecoin Signing Library - Rust / WASM / JSONRPC Signing Service | **In progress** |

Reviewer 2: filecoin-rs is ready to use, I believe. Ping `@jleni` on slack to double-check.
